### ISSUE TITLE
Can't use arrow keys and enter to select an item in the menu.

### ISSUE THREAD

**Issue by Pauan** [Pauan/tab-organizer#78]
Originally reported on Google Code with ID 76

```
What steps will reproduce the problem?
1. Open the context menu (Ctrl+M).
2. Try to use the arrow keys and enter to select an item.

What is the expected output? What do you see instead?
Can't. Should be able to.

Please provide any additional information below.
This could be tricky, we'll see...
```

Reported by `pcxunlimited` on 2010-10-06 23:17:52
- **Blocking**: #74

**Comment by Pauan** [comment]
```
This issue was updated by revision 1976b1e3fd.
```

Reported by `pcxunlimited` on 2010-10-12 03:31:24
- **Status changed**: `Pending`

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2010-10-17 17:04:03
- **Status changed**: `Verified`

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:44:48
- **Status changed**: `Fixed`
